packages/web/tests/components/DiscordCheck/DiscordButtonCheck.test.tsx

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
```

Comment:
The file name here should match the component exactly; make sure to rename this file and the parent directory